math256/src/tests.rs

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#[cfg(test)]
+mod tests {
+    use fluence_test::marine_test;
+
+    #[marine_test(config_path = "../Config.toml", modules_dir = "../artifacts/")]
+    fn add_test() {
+        assert_eq!(
```

Comment:
What about rewriting this and other multi-line tests in this file in the following way:
```rust
let actual = math256.add(
	"1000000000000000000000000000000000".to_string(),
	"1000000000000000000000000000000000".to_string()
).value;
let expected = "2000000000000000000000000000000000";

assert_eq!(actual, expected);
```

It's more readable, becase there is not so many identation levels.

Reply:
i do not see it as useful

Reply:
```
a=42
b=13
c = 123
c = a * b
```

vs

```
123 = 13 * 42
```

Reply:
It's not a right model, because the problem here is in a count of indentation levels. But since it's just a test and you have strongly commitment in this variant, let's leave it as it.